tests/test_all_rules.py

```diff
@@ -171,6 +171,22 @@ def test_max_signals_note(self):
                               f'Expected: {max_signal_standard_setup}\n\n'
                               f'Actual: {rule.contents.data.setup}')
 
+    def test_from_filed_value(self):
+        """ Add "from" Field Validation for All Rules"""
+        failures = []
+        valid_format = re.compile(r'^now-\d+[yMwdhHms]$')
+        for rule in self.all_rules:
+            from_field = rule.contents.data.get('from_')
+            if from_field is not None:
+                if not bool(valid_format.match(from_field)):
```

Comment:
another nitpick -- I think you don't need to convert to `bool` here, you can just check `if valid_format.match(from_field)` -- if there is no match, the method will return `None` which will be treated as false